src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AutoEnchanter.java

```diff
@@ -53,7 +53,7 @@ protected MachineRecipe findNextRecipe(BlockMenu menu) {
 
             // Check if the item is enchantable
             if (!isEnchantable(item)) {
```

Comment:
I just noticed that `isEnchantable` is missing an air check but that's just a side note.

Reply:
Do you want me to add that or can it be ignored?

Reply:
You can add it if you want but can also ignore it and we will add it later.
I approved it, so no explicit actions are expected nor required :)